site/src/lib/awsSes.ts

```diff
@@ -0,0 +1,48 @@
+import * as aws from "@aws-sdk/client-ses";
+import nodemailer from "nodemailer";
+import { convert } from "html-to-text";
+import { defaultProvider } from "@aws-sdk/credential-provider-node";
+import SESTransport from "nodemailer/lib/ses-transport";
+import { isProduction } from "./config";
+import { mustGetEnvVar } from "../util/api";
+
+const FROM_ADDRESS = "Support <noreply@blockprotocol.org>";
```

Comment:
How about moving this value to an env var too? Alternatively, can we specify the sender on the AWS end?

Reply:
Can create an environment variable, the reason I didn't was because I can't envision this value changing before we deprecate the Next API, unless I've missed something?

Reply:
I think it's fine as-is - we don't expect anyone to host their own version of this site so we don't need to account for as much configuration as we do for HASH.